src/integrationTest/java/uk/gov/hmcts/reform/em/hrs/config/TestAzureStorageConfiguration.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.em.hrs.config;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobContainerClientBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+@Configuration
+public class TestAzureStorageConfiguration {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestAzureStorageConfiguration.class);
+
+    private static final String AZURITE_IMAGE = "mcr.microsoft.com/azure-storage/azurite";
+    private static final int MAPPER_PORT = 10000;
+
+    private static final String ACCOUNT_NAME = "devstoreaccount1";
+    private static final String ACCOUNT_KEY =
+        "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==";
```

Comment:
is this the real account key? if so, it should probably come from a keyvault

Reply:
No, it's not.  This config starts up an azurite docker image for test purposes. These are the connection details for that azurite docker image